### PR TITLE
Code Documentation

### PR DESCRIPTION
**Pull request by hamzahass**
Added documentation for : 

- AddItemViewModel
- ChatViewModel
- QrCodeScannerViewModel
- User
- Item

### CODE REVIEW

**Review by Mouniraki — APPROVED** [review]
Good job, for some comments I see that you didn't explain the arguments of the function, I think that you should still put some more explanations to be more correct. Other than that, don't forget to assign yourself to the PR :)

File: app/src/main/java/com/github/brugapp/brug/view_model/AddItemViewModel.kt [hunk 1]
```diff
@@ -6,6 +6,13 @@ import androidx.lifecycle.ViewModel
 
 class AddItemViewModel : ViewModel() {
 
+    /**
+     * Verifies the information that the user entered for the new item
```
Comment (pmcintyr) [inline app/src/main/java/com/github/brugapp/brug/view_model/AddItemViewModel.kt:10]: Good job with the KDOC formatting here

File: app/src/main/java/com/github/brugapp/brug/model/Item.kt [hunk 2]
```diff
@@ -23,26 +23,44 @@ class Item(
     private var itemID: String = ""
     private var lastLocation: LocationService? = null
 
+    /**
+     * returns the item's last location
```
Comment (pmcintyr) [inline app/src/main/java/com/github/brugapp/brug/model/Item.kt:27]: this function is nullable, it is a minor detail but if you want we can specify this in the documentation

File: app/src/main/java/com/github/brugapp/brug/view_model/ChatViewModel.kt [hunk 3]
```diff
@@ -467,13 +467,13 @@ class ChatViewModel : ViewModel() {
     }
 
     /**
-     * TODO @HAMZA
+     * Sends the audio message to the corresponding conversation
```
Comment (pmcintyr) [inline app/src/main/java/com/github/brugapp/brug/view_model/ChatViewModel.kt:470]: good explanations once more!

**Review by pmcintyr — APPROVED** [review]

File: app/src/main/java/com/github/brugapp/brug/model/Item.kt (hunk 2 above)
Comment (hamzahass) [reply app/src/main/java/com/github/brugapp/brug/model/Item.kt:27]: Okay thanks, for pointing this out

**Pull request merged**